requirements.txt

```diff
@@ -13,10 +13,7 @@ torchvision
 transformers>=4.31.0
 tokenizers>=0.13.3
 spacy>=2.3
-
-# https://github.com/yaml/pyyaml/issues/601
-PyYAML>=3.12,<6.0.1
-
+PyYAML>=3.12,<6.0.1,!=5.4.*
```

Comment:
nit: Since we're excluding a very small range of PyYAML versions, could you add a short comment explaining why 5.4.x is excluded?